docs/modules/technical-guide/pages/common-concepts/application.adoc

```diff
@@ -0,0 +1,41 @@
+= Application
+
+include::common:partial$_header.adoc[]
+
+//-----------------------------------------------------------------------------
+// This file describes the application used to bootstrap Scout
+//-----------------------------------------------------------------------------
+
+This section describes the module structure of a Scout application and how the bootstrap is initiated.
+
+== Module structure
+
+For each server (i.e. single server for Scout JS and UI/backend server for Scout Classic application), the following modules are used to build or run the application:
+
+* `.app`: main module serving as base for all dependencies
+** `.app.dev`: used for development
+** `.app.image`: used for xref:getstarted:run-as-docker-container.adoc[docker image build]
+** `.app.zip`: used for xref:getstarted:run-as-service.adoc[deployment via shell scripts]
+
+Each leaf module (`.dev`, `.image`, `.zip`) contains a `config.properties`, `logback.xml` and possible other resources relevant for the corresponding build/application run.
+Dependencies only relevant for a certain build (e.g. development) must only be added in the corresponding module.
+Dependencies relevant for all builds should be placed in the `.app` module.
+
+== Bootstrap Class
+
+`org.eclipse.scout.rt.app.Application` is the main class to bootstrap a Scout application.
+This is the class started via run configuration in your IDE as well as the one used within the Docker Image (`.app.image`) or the shell scripts within the ZIP build (`.app.zip`).
+
+Besides starting the xref:technical-guide:common-concepts/platform.adoc#sec-app.lifecycle[platform], the main class takes care of running an embedded Jetty application server, so that your application is accessible via browser.
+
+== Servlet (Filter) Contributors
+
+Own servlets or servlet filters are registered by providing an implementation of:
+
+* `org.eclipse.scout.rt.jetty.IServletContributor`
+* `org.eclipse.scout.rt.jetty.IServletFilterContributor`
+
+This is the programmatic equivalent to a registration via `web.xml` as used for Tomcat for example.
```

Comment:
Bumpy sentence, maybe a suggestion (but do not even know if the suggestion is the perfect sentence yet):

```suggestion
This is the programmatic equivalent to a registration via `web.xml` which was used in older versions for Tomcat application servers for example.
```

Reply:
Replaced by:
For .war deployments on an application servers (e.g. Tomcat) a servlet (filter) registration is configured via `web.xml`.
The registration described above is the programmatic equivalent for the Scout application.